src/http/nodegen/middleware/handleHttpException.ts

```diff
@@ -4,19 +4,38 @@ import { HttpException } from '../errors';
 import NodegenRequest from '../interfaces/NodegenRequest';
```

Comment:
personally, I'd almost rather a complete different middleware than just passing options and doing if this then that otherwise something else - again, would like to push this to a separate cleanup merge though just to get this through.